### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by haoweicrushu**
fix typo

### CODE REVIEW

File: pkg/git/commit_list_builder.go [hunk 1]
```diff
@@ -177,7 +177,7 @@ func (c *CommitListBuilder) getNormalRebasingCommits() ([]*commands.Commit, erro
 func (c *CommitListBuilder) getInteractiveRebasingCommits() ([]*commands.Commit, error) {
 	bytesContent, err := ioutil.ReadFile(fmt.Sprintf("%s/rebase-merge/git-rebase-todo", c.GitCommand.DotGitDir))
 	if err != nil {
-		c.Log.Info(fmt.Sprintf("error occured reading git-rebase-todo: %s", err.Error()))
+		c.Log.Info(fmt.Sprintf("error occurred reading git-rebase-todo: %s", err.Error()))
```
Comment (mjarkk) [inline pkg/git/commit_list_builder.go]: @jesseduffield isn't this supposed to be inside the translations files?

Comment (jesseduffield) [reply pkg/git/commit_list_builder.go]: I think logs were okay to not be translated, because we weren't expecting end-users to read them

Comment (mjarkk) [reply pkg/git/commit_list_builder.go]: Ok then it's probably useless to move the location of this string.

**Comment by jesseduffield** [comment]
Thanks for making this :)

**Pull request merged**